ubuntu_desktop_installer/pubspec.yaml

```diff
@@ -25,7 +25,7 @@ dependencies:
     sdk: flutter
   flutter_localizations:
     sdk: flutter
-  intl:
+  intl: ^0.17.0
```

Comment:
Should be: intl: ^0.17.0-nullsafety.2

Reply:
my bad, I thought it was obsolete due to newer version, I was wrong